spec/fixtures/api/gpu-info.js

```diff
@@ -0,0 +1,17 @@
+const { app } = require('electron')
+
+app.commandLine.appendSwitch('--disable-software-rasterizer')
```

Comment:
Can you try to run CI jobs without this flag to check if all tests fail exactly as expected?